test/WebSites/Directory.Build.props

```diff
@@ -3,9 +3,8 @@
   <Import Project="..\..\Directory.Build.props" />
 
   <PropertyGroup>
-    <DeveloperBuildTestWebsiteTfms>netcoreapp2.2</DeveloperBuildTestWebsiteTfms>
+    <DeveloperBuildTestWebsiteTfms>netcoreapp3.0</DeveloperBuildTestWebsiteTfms>
     <StandardTestWebsiteTfms>$(DeveloperBuildTestWebsiteTfms)</StandardTestWebsiteTfms>
     <StandardTestWebsiteTfms Condition=" '$(DeveloperBuild)' != 'true' ">$(StandardTestWebsiteTfms)</StandardTestWebsiteTfms>
-    <StandardTestWebsiteTfms Condition=" '$(DeveloperBuild)' != 'true' AND '$(OS)' == 'Windows_NT' ">$(StandardTestWebsiteTfms);net461</StandardTestWebsiteTfms>
```

Comment:
Build time compilation doesn't work very well with desktop and I had a couple of other issues when I tried treating it the same as RCL. I'm going to remove desktop from the mix for now since it doesn't seem particularly relevant in 3.0

Reply:
Tracking this via https://github.com/aspnet/Identity/issues/2046